src/modules/bundle.ts

```diff
@@ -122,10 +145,22 @@ export class CollectionModule extends ModuleWithRoles<NFTCollectionContract> {
     );
   }
 
-  public async isApproved(address: string, operator: string): Promise<boolean> {
+  public async isApproved(address: string, operator: string, tokenContract?: string, tokenId?: BigNumberish): Promise<boolean> {
```

Comment:
I'm confused, what's this `tokenContract` about? 
This is public interfaces, if we want to do it for our internal convenience we should move them to private functions.

Reply:
@ayshptk we should change this for sure.